css/index.css

```diff
@@ -143,4 +186,43 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
```

Comment:
The .middle-content .boxes .box CSS in the index.css file is where you want to make use of parent/child relationships to style the boxes.